HW9_2.cpp

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>

```

Comment:
don't undrestand what number it is. Also I want input to be implemented for arbitrary dimensions